CONTRIBUTING.md

```diff
@@ -4,13 +4,13 @@ Thank you for your interest in contributing to Coq! There are many ways to contr
 
 ## Bug Reports
 
-Bug reports are enormously useful to identify issues with Coq; we can't fix what we don't know about. To report a bug, please open an issue on the [Coq GitHub repository](https://github.com/coq/coq/issues) (you'll need a GitHub account). You can file a bug for any of the following:
+Bug reports are enormously useful to identify issues with Coq; we can't fix what we don't know about. To report a bug, please open an issue in [our issue tracker](https://github.com/coq/coq/issues) (you'll need a GitHub account). You can file a bug for any of the following:
```

Comment:
I would prefer `in the [Coq issue tracker]` (to make both references consistent)